.github/workflows/mac-wpt.yml

```diff
@@ -43,9 +43,12 @@ jobs:
           python3 ./mach bootstrap
       - name: Smoketest
         run: python3 ./mach smoketest --release
+      - name: WPT legacy layout argument
+        if: inputs.layout == 'layout-2013'
+        run: echo "WPT_COMMAND_LINE_ARG=--legacy-layout" >> $GITHUB_ENV
```

Comment:
this could be reduced to something like:
```yaml
ENV:
    WPT_COMMAND_LINE_ARG: ${{ inputs.layout == 'layout-2013' && "--legacy-layout" || "" }}
```

which should be faster